teamwork/apps/projects/views/EditProjectView.py

```diff
@@ -115,7 +115,7 @@ def edit_project(request, slug):
         elif profAdded:
             messages.add_message(request, messages.SUCCESS, "Greppers have been added to the project.")
         else:
-            messages.add_message(request, messages.WARNING, "Failed to invite member(s) to project. Make sure they are enrolled in this course.")
+            messages.add_message(request, messages.WARNING, "Student(s) is already added to the project.")
```

Comment:
This message will need a check to see what failed since this message can also still be triggered if the user is not enrolled in the course

Reply:
I think we added an elif statement before but it was giving us and error, is there another approach we should use

Reply:
I would recommend just putting another if within the else that determines which message to send

Reply:
I would put a new if inside of the else that determines which message to send